donate/payments/views.py

```diff
@@ -104,29 +104,27 @@ def queue_ga_transaction(self, id, currency, amount, name, category):
         queue_ga_event(self.request, ['ecommerce:send'])
 
     def queue_datalayer_transaction(self, id, currency, amount, frequency, payment_method="credit card"):
-        queue_data_layer_event(
+        queue_datalayer_event(
```

Comment:
Updated the name of this function, as well as updating the data sent to be just an object, instead of an object wrapped in []